src/yang/ietf-bgp-rib-types.yang

```diff
@@ -53,6 +53,7 @@ submodule ietf-bgp-rib-types {
        local router as originator";
   }
 
+  // XXX JMH - this is really just another flavor of invalid-as-loop.  Likely should drop.
```

Comment:
Ok. Will drop it.

Reply:
Open issue. Cisco probably does it. Check with Jacob.

Reply:
Turns out that RFC 7854 does define this type - Stat Type = 6: (32-bit Counter) Number of updates invalidated due
      to AS_CONFED loop.